src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -100,6 +100,7 @@ import GraphicsLayerWrapper from "./graphicsLayer";
 import Popup from "@arcgis/core/widgets/Popup";
 import ElevationLayer from "@arcgis/core/layers/ElevationLayer";
 import PopupWidgetWrapper from "./popupWidgetWrapper";
+import PortalItem from "@arcgis/core/portal/PortalItem";
```

Comment:
one question.  PortalItem has been operational as part of the mapview up until this last week.  The adjustment to the layer class created a problem with the existing Portalitem reference here, which required a reworking of the code and an import statement here.  Do you see the need to continue to add these imports which are already apart of different classes into the arcGisJSInterop imports as the code continues to evolve?

Reply:
This is an import, same as a `using` in C#. In this case, it's unused, I had added it while trying out something, and then deleted the calling code. It's not harmful, but can be removed (I will clean it up). 

The reason it's not necessary is that ArcGIS does a _lot_ of auto-casting objects to defined types. So if you look at the uses of `PortalItem` in this file, they are usually just declared as generic objects like so:
```javascript
{
    id: id
}
```